euth_wagtail/assets/scss/components/_dashboard.scss

```diff
@@ -174,9 +182,37 @@ h1.dashboard-title {
     }
 
     .blueprint-phases {
-        &-item {
-            max-width: 30%;
-            vertical-align: top;
+        ul {
+            display: inline-block;
+        }
+
+        li {
+            overflow: hidden;
+            @include rem(padding-bottom, 10px);
+            @include rem(width, 300px);
+        }
+
+        .icon {
+            float: left;
+            @include rem(padding-bottom, 4px);
+            @include rem(padding-right, 10px);
+        }
+
+        .description {
+            text-align: left;
+        }
+
+        .circled {
```

Comment:
I think the entire icon can be done with a lot less markup and magic values using font awesome stacked items. But that is just a personal opinion. Any how it seems to me that the icons are also not always vertically centered.

![](https://image.prntscr.com/image/-0_x0nocTg6BjPla_tkgkQ.png)

But I am not sure if that can be fixed. Maybe we need another symbol.